buildflow/runtime/ray_io/sqs_io.py

```diff
@@ -32,7 +33,7 @@ class SQSSource(io.StreamingSource):
     def get_boto_client(self):
         if self._test_sqs_client is not None:
             return self._test_sqs_client
-        return boto3.client('sqs')
+        return boto3.client(service_name='sqs', region_name=self.region)
```

Comment:
nit: If region_name is optional in boto3.client, maybe make it optional up above

Reply:
good call, then users can still set it via environment variables if they want